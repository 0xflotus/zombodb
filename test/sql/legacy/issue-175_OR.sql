alter index idxso_posts set (options='user_data:(owner_user_id=<public.so_users.idxso_users>id), comment_data:(id=<public.so_comments.idxso_comments>post_id)');
select count(*) from so_posts where so_posts==> 'a or b';
alter index idxso_posts reset (options);
