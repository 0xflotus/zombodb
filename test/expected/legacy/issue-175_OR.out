alter index idxso_posts set (options='user_data:(owner_user_id=<public.so_users.idxso_users>id), comment_data:(id=<public.so_comments.idxso_comments>post_id)');
select count(*) from so_posts where so_posts==> 'a or b';
 count  
--------
 158579
(1 row)

alter index idxso_posts reset (options);
